crates/noirc_evaluator/src/ssa_refactor/ir/dfg.rs

```diff
@@ -189,6 +194,11 @@ impl DataFlowGraph {
         self.values.insert(Value::Function(function))
     }
 
+    /// Gets or creates a ValueId for the given FunctionId.
+    pub(crate) fn import_foreign_function(&mut self, function: &str) -> ValueId {
+        self.values.insert(Value::ForeignFunction(function.to_owned()))
```

Comment:
Comment needs to be updated.

We should also have an enumeration for Oracles. It is unclear without it which values may be valid for this string.

Reply:
Will update comment as part of #1643 refactor. An enumeration would be good for `nargo` oracle builtins such as println, however, we want to let developers specify their own oracles if they so chose. Need to think on the best way to include these two flows

Reply:
Ah, in that case we should likely give them a separate BrilligID like we do for FunctionIds